### PR TITLE
Added single reference code support

### PR DESCRIPTION
**Pull request by jbradl11**
Added support for single reference codes. 

Jira Test: https://jira.mitre.org/browse/BONNIE-918

### CODE REVIEW

**Comment by lukoktonos** [comment]
Regression test passed.

**Comment by hossenlopp** [comment]
Travis is failing because the gemfile.lock changes were not included here.

File: lib/measures/loading/cql_loader.rb [hunk 1]
```diff
@@ -53,15 +53,90 @@ def self.load_mat_cql_exports(user, file, out_dir, measure_details, vsac_user, v
...
+      # Add all single code references from each elm file
+      elms.each do | elm |
+        # Check if elm has single reference code.
+        if elm['library'] && elm['library']['codes'] && elm['library']['codes']['def']
+          # Loops over all single codes and saves them as fake valuesets.
+          elm['library']['codes']['def'].each do |code_reference|
+            code_sets = {}
+            code_system = code_reference['codeSystem']['name']
+
+            # TODO: Will it always be structured as "LOINC:2.4"?
```
Comment (lukoktonos) [inline lib/measures/loading/cql_loader.rb:115]: I have one showing as:
image

**Review by lukoktonos — APPROVED** [review]
JIRA and Regression tests passed. Looks good.

**Review by holmesie — APPROVED** [review]
Tested, looks good

**Pull request merged**